airbyte-webapp/src/types/react-table.d.ts

```diff
@@ -7,7 +7,14 @@ import "@tanstack/react-table";
  */
 declare module "@tanstack/table-core" {
   interface ColumnMeta {
-    thClassName?: string;
+    collapse?: boolean;
+    customPadding?: {
```

Comment:
I'd prefer to avoid adding style props here since after some time we will get the same result as we have in the current implementation.